main.go

```diff
@@ -45,56 +43,19 @@ const (
 	junitFlagKey                  = "junit"
 	TNFReportKey                  = "cnf-certification-test"
 	extraInfoKey                  = "testsExtraInfo"
-	noLabelsExpr                  = "none"
 	logFileName                   = "cnf-certsuite.log"
 	logFilePermissions            = 0o644
 )
 
-const (
-	labelsFlagName         = "label-filter"
-	labelsFlagDefaultValue = "common"
-
-	labelsFlagUsage = "--label-filter <expression>  e.g. --label-filter 'access-control && !access-control-sys-admin-capability'"
-
-	timeoutFlagName         = "timeout"
-	timeoutFlagDefaultvalue = 24 * time.Hour
-
-	timeoutFlagUsage = "--timeout <time>  e.g. --timeout 30m  or -timeout 1h30m"
-
-	listFlagName         = "list"
-	listFlagDefaultValue = false
-
-	listFlagUsage = "--list Shows all the available checks/tests. Can be filtered with --label-filter."
-
-	serverModeFlagName         = "serverMode"
-	serverModeFlagDefaultValue = false
-
-	serverModeFlagUsage = "--serverMode or -serverMode runs in web server mode."
-)
-
 var (
 	claimPath *string
```

Comment:
Can't move this one to the flags package?

Reply:
fixed